### ISSUE TITLE
'whitePoints' calculation in GameResult.scala  maybe wrong ?

### ISSUE THREAD

**Issue by javadev-berlin** [maxpumperla/ScalphaGoZero#21]
In line 42 of [GameResult.scala ](https://github.com/maxpumperla/ScalphaGoZero/blob/master/src/main/scala/org/deeplearning4j/scalphagozero/scoring/GameResult.scala) you have 
```javascript
 val whitePoints: Int = numWhiteStones + numWhiteStones
```
Shouldn't it be 

```javascript
  val whitePoints: Int = numWhiteTerritory + numWhiteStones
```
?

**Comment by maxpumperla** [comment]
@javadev-berlin of course, thank you! mind sending a PR?

**Comment by barrybecker4** [comment]
This was fixed a long time ago in revision ea09435fc561cf5891d924245b87b908ac019154.
